README.rst

```diff
@@ -10,11 +10,9 @@ PyAnsys Actions
    :target: https://opensource.org/licenses/MIT
    :alt: MIT
 
-A repository containing a collection of `GitHub Workflows`_ to be reused by
-projects in the `PyAnsys` ecosystem.
+A repository containing a collection of `GitHub Workflows
+<https://docs.github.com/en/actions/using-workflows/about-workflows>`_ to be
+reused by projects in the PyAnsys ecosystem.
 
-
-.. LINKS AND REFERENCES
-
-.. _GitHub workflows: https://docs.github.com/en/actions/using-workflows/about-workflows
-.. _pyansys: https://docs.pyansys.com/
+Refer to `actions.docs.pyansys.com <https://actions.docs.pyansys.com>`_ for more
+information about available actions and how to use those.
```

Comment:
```suggestion
For more information on available actions and how to use them, see `actions.docs.pyansys.com <https://actions.docs.pyansys.com>`_ .
```
Style guidelines say to start with why they would go see a resource and to use "on" as the preposition.

Reply:
Taking notes on this, thanks for remembering this, Kathy.